src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final int DIVIDER_FOR_AREA = 2;
+    private int bothside;
```

Comment:
The variable name `bothside` is misleading, as it suggests that both sides of the rectangle are of equal length, which is not necessarily the case for a rectangle. Consider using more descriptive names like `width` and `height` to represent the sides of the rectangle.